index.html

```diff
@@ -1,31 +1,40 @@
 <!doctype html>
 <html>
+
   <head>
-    <link rel='stylesheet' href='styles.css'>
+    <title>Jesse Jams</title>
+    <link rel='stylesheet' href='styles.css' type'text/css'>
   </head>
+
   <body>
+
     <header>
       <h1>The Header</h1>
     </header>
+
     <aside>
       <h2>A Side Bar</h2>
     </aside>
+
     <div class='main'>
       <h2>The Main Content</h2>
-      <div class='left'>
+      <div class='panel'>
```

Comment:
A semantic tag would be something like `<main>`. While this classname is a bit more semantic, the question is looking for a new HTML tag.